### PR TITLE
Updated symbol masterExec not to use 'master' keyword, and local vari…

### LINKED ISSUE #94901: Update test/e2e/framework to use the term "Control Plane"
There are a number of variable and functions that can outright be renamed to refer to "control plane" instead of "master".  There are also some flags which should be deprecated (and corresponding changes made in jobs)

```
$ ag -l master test/e2e/framework | sort
test/e2e/framework/config/config.go
test/e2e/framework/framework.go
test/e2e/framework/gpu/gpu_util.go
test/e2e/framework/ingress/ingress_utils.go
test/e2e/framework/kubelet/stats.go
test/e2e/framework/kubesystem/kubesystem.go
test/e2e/framework/log_size_monitoring.go
test/e2e/framework/network/utils.go
test/e2e/framework/node/resource.go
test/e2e/framework/node/wait.go
test/e2e/framework/node/wait_test.go
test/e2e/framework/nodes_util.go
test/e2e/framework/providers/gce/firewall.go
test/e2e/framework/resource_usage_gatherer.go
test/e2e/framework/skipper/skipper.go
test/e2e/framework/ssh/ssh.go
test/e2e/framework/test_context.go
test/e2e/framework/util.go
```

/kind cleanup
/wg naming
/sig testing
/area test

### PR DESCRIPTION
**Pull request by vishsangale**
…able names in prepareData

**What type of PR is this?**
/kind cleanup
/wg naming
/sig testing
/area test

**What this PR does / why we need it**:
Update references to word master by control plane or instance
**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
n this review, I have only changed one symbol `masterExec` and the comments related to it. As well as local variables in the function `prepareData`
**Does this PR introduce a user-facing change?**:

```release-note

```
/release-note-none
**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/apimachinery/etcd_failure.go
Comment (spiffxp) [inline test/e2e/apimachinery/etcd_failure.go:101]: would a better term be etcdInstanceExec?

File: test/e2e/framework/log_size_monitoring.go
Comment (spiffxp) [inline test/e2e/framework/log_size_monitoring.go:134]: instance of what?

**Review by spiffxp — CHANGES_REQUESTED** [review]

**Comment by spiffxp** [comment]
/hold
ref: https://github.com/kubernetes/kubernetes/issues/94901#issuecomment-702344182

**Comment by fedebongio** [comment]
/assign @lavalamp 
it only needs approval, thanks!

File: test/e2e/apimachinery/etcd_failure.go
Comment (lavalamp) [reply test/e2e/apimachinery/etcd_failure.go:101]: controlPlaneInstance

**Comment by lavalamp** [comment]
heh, sorry, this is the first time I saw this in my email

**Comment by spiffxp** [comment]
If you want to rebase and address review comments then it's ok to /reopen this. Otherwise will identify this as a change someone else can take on with a new PR

**Pull request closed without merging**
